docs/development/vscode.md

```diff
@@ -0,0 +1,80 @@
+# Setting Up VS Code for Debugging Zulip
+
+## Overview
+
+Set up VS Code to debug the django zulip sever. The following can only
+setup VS Code to debug the *django application* and not the *JS*
+(frontend) part of zulip. This is still a work in progress and any
+suggestions to simplify or ease the workflow are encouraged.
+
+I hope this helps the community of VS Code users working on zulip and
+helps them significantly ! :D
+
+## Steps
+
+1. Download and install VS Code.
+2. Download and install the
+   [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
+   extension for VS Code by Microsoft.  
```

Comment:
This part can be automated. Here's our `.vscode/extensions.json` file in the mobile repo:
```json5
{
	// See http://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp

	// List of extensions which should be recommended for users of this workspace.
	"recommendations": [
		"msjsdiag.vscode-react-native",
		"flowtype.flow-for-vscode",
		"dbaeumer.vscode-eslint",
		"esbenp.prettier-vscode"
	],

	// List of extensions recommended by VS Code that should not be recommended for users of this workspace.
	// Nothing in here but adding this in order to add extentions later as required.
	"unwantedRecommendations": []
}
```

Reply:
Wow thanks for this suggestion, will definitely look into this and add it here! @gnprice